pkg/combustion/registry.go

```diff
@@ -61,11 +61,18 @@ func configureRegistry(ctx *image.Context) ([]string, error) {
 		return nil, fmt.Errorf("configuring helm: %w", err)
 	}
 
-	if err = configureEmbeddedArtifactRegistry(ctx, helmTemplatePath, helmManifestHolderDir); err != nil {
+	configured, err := configureEmbeddedArtifactRegistry(ctx, helmTemplatePath, helmManifestHolderDir)
+	if err != nil {
 		log.AuditComponentFailed(registryComponentName)
 		return nil, fmt.Errorf("configuring embedded artifact registry: %w", err)
 	}
 
+	if !configured {
+		log.AuditComponentSkipped(registryComponentName)
+		log.Audit("Embedded Artifact Registry skipped: Provided manifests/helm charts contain no images.")
```

Comment:
I meant a log statement to the log file i.e. `zap.S()...` which we can later use. `log.AuditComponentSkipped(registryComponentName)` is more than enough for the user facing side.